mobile/bindings.go

```diff
@@ -0,0 +1,47 @@
+//go:build mobile
+// +build mobile
+
+package mobile
+
+import (
+	"context"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/lightningnetwork/lnd/keychain"
+	"github.com/lightninglabs/lightning-node-connect/mailbox"
+	"google.golang.org/grpc"
+)
+
+// connectMailbox tries to establish a connection through LNC using the given
+// connect phrase and the mailbox server.
+func ConnectMailbox(ctx context.Context,
+	connectPhrase []string, mailboxServerAddr string) (grpc.ClientConnInterface, error) {
+
+	var mnemonicWords [mailbox.NumPassphraseWords]string
+	copy(mnemonicWords[:], connectPhrase)
+	passphrase := mailbox.PassphraseMnemonicToEntropy(mnemonicWords)
+
+	privKey, err := btcec.NewPrivateKey()
+	if err != nil {
+		return nil, err
+	}
+	ecdh := &keychain.PrivKeyECDH{PrivKey: privKey}
+
+	connData := mailbox.NewConnData(ecdh, nil, passphrase[:], nil, nil, nil)
```

Comment:
hmmm, this will only work for the first connection... after the first connection, the local & remote static keys are required

Reply:
unless of course you specify in the dial options that only Handshake V1 should be used.... but i dont suggest you do that unless there is a strong reason to